Source/Lib/Codec/EbEncHandle.c

```diff
@@ -2521,13 +2520,13 @@ static EB_ERRORTYPE VerifySettings(\
 
     if (config->hrdFlag > 1)
     {
-        printf("Error Instance %u: hrdFlag must be [0 - 1]\n", channelNumber + 1);
+        printf("SVT [Error]: Instance %u: hrdFlag must be [0 - 1]\n", channelNumber + 1);
         return_error = EB_ErrorBadParameter;
```

Comment:
please use "SVT_LOG("SVT [Error]:"

Reply:
Please follow the coding style here. 
if () {
}

Reply:
Modified and pushed (Commit: 9db1593)

Reply:
Please fix the indent of your code changes. Use space instead of tab.